pkg/operator/ceph/object/user/controller.go

```diff
@@ -493,10 +494,13 @@ func updateStatus(client client.Client, name types.NamespacedName, status string
 		return
 	}
 	if user.Status == nil {
-		user.Status = &cephv1.Status{}
+		user.Status = &cephv1.ObjectStoreUserStatus{}
 	}
 
 	user.Status.Phase = status
+	if user.Status.Phase == k8sutil.ReadyStatus {
+		user.Status.Info = map[string]string{"secretName": generateCephUserSecretName(user)}
```

Comment:
Please create a function that returns the map for `user.Status.Info` instead. So if we want to add more in the future it'll be easier.

Reply:
> Please create a function that returns the map for `user.Status.Info` instead. So if we want to add more in the future it'll be easier.

Do we want another function that will return the map as we already have one function that returns the secret name of string type?

Reply:
Yes.

Reply:
Done.